Tests/UnitTests/Paywalls/Events/PaywallEventsBackendTests.swift

```diff
@@ -72,7 +72,7 @@ private extension BackendPaywallEventTests {
         offeringIdentifier: "offering_1",
         paywallRevision: 5,
         sessionID: .init(uuidString: "98CC0F1D-7665-4093-9624-1D7308FFF4DB")!,
-        displayMode: .condensedFooter,
+        displayMode: .fullScreen,
```

Comment:
`.fullScreen` is the only available on in `watchOS`, so I changed this to fix those tests.